### PR TITLE
Change `SyncInterval` to a data class

### PR DESCRIPTION
**Pull request by vinaysshenoy**
https://app.clubhouse.io/simpledotorg/story/4148/remove-separation-between-daily-and-frequent-syncs

### CODE REVIEW

File: app/src/androidTest/java/org/simple/clinic/di/TestSyncConfigModule.kt [hunk 1]
```diff
@@ -7,15 +7,26 @@ import org.simple.clinic.sync.SyncConfigType
...
+import java.time.Duration
 
 @Module
 class TestSyncConfigModule {
 
+  @Provides
+  fun syncInterval(): SyncInterval {
+    return SyncInterval(
+        frequency = Duration.ofMinutes(16),
+        backOffDelay = Duration.ofMinutes(5L)
```
Comment (Honey14) [inline app/src/androidTest/java/org/simple/clinic/di/TestSyncConfigModule.kt:19]: What is 5L minutes?

**Review by Honey14 — APPROVED** [review]
Looks good. Left one question, PTAL

File: app/src/androidTest/java/org/simple/clinic/di/TestSyncConfigModule.kt (hunk 1 above)
Comment (vinaysshenoy) [reply app/src/androidTest/java/org/simple/clinic/di/TestSyncConfigModule.kt:19]: Ah the `L` suffix means the type is a `Long`. The `ofMinutes` method takes a `Long` as a parameter.

**Pull request merged**